common/persistence/sql/sqlHistoryManager.go

```diff
@@ -440,10 +446,48 @@ func (m *sqlHistoryV2Manager) GetAllHistoryTreeBranches(
 	ctx context.Context,
 	request *p.GetAllHistoryTreeBranchesRequest,
 ) (*p.GetAllHistoryTreeBranchesResponse, error) {
-
-	// TODO https://github.com/uber/cadence/issues/2458
-	// Implement it when we need
-	panic("not implemented yet")
+	page := historyTreePageToken{
+		ShardID:  0,
+		TreeID:   serialization.MustParseUUID(minUUID),
```

Comment:
Can we replace this with empty binary?
I think It's lower than any other non-empty. 
```
mysql> select * from test_binary order by bid;
+------------------+
| bid              |
+------------------+
|                  |
| 0                |
| a                |
| b                |
+------------------+
4 rows in set (0.01 sec)

mysql> show create table test_binary;
+-------------+----------------------------------------------------------------------------------------------------------------------+
| Table       | Create Table                                                                                                         |
+-------------+----------------------------------------------------------------------------------------------------------------------+
| test_binary | CREATE TABLE `test_binary` (
  `bid` binary(16) NOT NULL,
  PRIMARY KEY (`bid`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 |
+-------------+----------------------------------------------------------------------------------------------------------------------+
```